packages/grid/src/features/standard-table/stories/StandardTable.stories.tsx

```diff
@@ -293,3 +309,35 @@ export const SomeExpandableRows = () => {
   });
   return <StandardTable items={items} config={config} />;
 };
+
+export const WithStickyTableHeader = () => {
+  const config = createConfig(undefined, undefined, {
```

Comment:
Is it a good idea to use abstactions in stories?

Reply:
Why not?
It is pretty much the same as mock creator.

Reply:
This code does not represent how you would use the component in your own app:

![image](https://user-images.githubusercontent.com/817046/98090535-6d1c0000-1e84-11eb-9554-3775cb3c608d.png)

Reply:
Please be more specific.